src/Application/FileProcessor.php

```diff
@@ -143,9 +149,10 @@ private function printFile(File $file, Configuration $configuration, string $fil
              * On very first content level
              */
             $originalFileContent = $file->getOriginalFileContent();
-            $ltrimOriginalFileContent = ltrim($originalFileContent);
+            $ltrimOriginalFileContent = preg_replace(self::OPEN_TAG_SPACED_REGEX, '<?php', ltrim($originalFileContent));
+            $ltrimNewContent = preg_replace(self::OPEN_TAG_SPACED_REGEX, '<?php', ltrim($newContent));
 
-            if ($ltrimOriginalFileContent === $newContent) {
+            if ($ltrimOriginalFileContent === $ltrimNewContent) {
```

Comment:
the origininal compare need to be used first, compare regex version later as regex can be expensive

Reply:
Good point, splitting it out to 2 steps now. Sorry for the delay there btw.